config/locales/gobierto_participation/views/es.yml

```diff
@@ -22,6 +22,29 @@ es:
           donde debatiremos en persona sobre los temas de este proceso. Si no puedes
           asistir, iremos publicando online todo lo que se hable en la reuniones para
           facilitar el seguimiento y hacer accesibles las conclusiones.
+    issues:
+      current_group:
+        current_stage: Fase actual
+      index:
+        intro: Tus ideas ayudan a mejorar %{location_name} y la vida de todos los
```

Comment:
This should be `intro_html` to don't escape the HTML:

![screen shot 2017-08-18 at 09 00 50](https://user-images.githubusercontent.com/17616/29447781-bed377c2-83f3-11e7-91c4-2a67b4264f55.png)